Find a functionally equivalent code:android.widget.ActivityChooserView.updateAppearance:COMMENT
Method Modifier: private     
Comment:/**
 * Updates the buttons state.
 */

@@ -23,30 +23,30 @@
     }
     // Activity chooser content.
     if (mDefaultActivityButton.getVisibility() == VISIBLE) {
-        mActivityChooserContent.setBackgroundDrawable(mActivityChooserContentBackground);
+        mActivityChooserContent.setBackground(mActivityChooserContentBackground);
     } else {
-        mActivityChooserContent.setBackgroundDrawable(null);
+        mActivityChooserContent.setBackground(null);
     }
 }

